README.md

```diff
@@ -2,6 +2,15 @@
 
 Base Node.js image that contains common dependencies used in our organization.
 
+## Adding an image
+
+1. Create directory for image
+2. Add Dockerfile and any related policies to directory
+3. Add a target for the image to the Makefile: [Example](https://github.com/articulate/docker-articulate-node/pull/70/commits/c6e1f85dd8c472b15029899277ebe05956864708#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R7-R9)
```

Comment:
The link in the example doesn't include adding the actual make target. That was done [here](https://github.com/articulate/docker-articulate-node/pull/70/commits/3d58f80b8be0da9edcd1e38d5ffe600186199ac9#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R33).

We could maybe just merge this step and the one below into a generic "update the makefile"